src/linter.cc

```diff
@@ -265,7 +325,7 @@ LinterResult ASTNodeRule::ApplyTo(absl::string_view sql,
   while (!is_the_end) {
     status = ParseNextScriptStatement(&location, ParserOptions(), &output,
                                       &is_the_end);
-    if (!status.ok()) return LinterResult(status);
+    if (!status.ok()) return LinterResult();
```

Comment:
Why was status removed from LinterResult input argument?

Reply:
That was the bug. The status error is already given by parser check, so it is just dublicate. When parser is disabled by nolint, it will also be disabled since it is parser dependant.